packages/react-instantsearch/src/widgets/LookingSimilar.tsx

```diff
@@ -6,7 +6,7 @@ import type {
   LookingSimilarProps as LookingSimilarPropsUiComponentProps,
   Pragma,
 } from 'instantsearch-ui-components';
-import type { AlgoliaHit, BaseHit, Hit } from 'instantsearch.js';
+import type { AlgoliaHit, BaseHit } from 'instantsearch.js';
```

Comment:
now that it has Hit in the connector, the four React InstantSearch widgets should use exclusively Hit I believe.